ledger-tool/src/main.rs

```diff
@@ -564,15 +565,22 @@ fn load_bank_forks(
             compression: CompressionType::Bzip2,
         })
     };
+    let blockstore = open_blockstore(&ledger_path, access_type);
     let account_paths = if let Some(account_paths) = arg_matches.value_of("account_paths") {
         account_paths.split(',').map(PathBuf::from).collect()
     } else {
-        vec![ledger_path.join("accounts")]
+        if blockstore.is_primary_access() {
+            vec![ledger_path.join("accounts")]
+        } else {
+            let non_primary_accounts_path = ledger_path.join("accounts.ledger-tool");
+            eprintln!("Default accounts path is switched aligning with Blockstore's non-primary access: {:?}", non_primary_accounts_path);
```

Comment:
TODO: maybe `warn!` to aligh with other messages: https://github.com/solana-labs/solana/pull/10209#issuecomment-633203847